src/components/common/BaseMoney.vue

```diff
@@ -19,7 +16,10 @@ export default {
     },
 
     amount() {
-      return this.money.centAmount / (10 ** this.money.fractionDigits);
+      if (this.money) {
+        return this.money.centAmount / (10 ** this.money.fractionDigits);
+      }
+      return 0;
```

Comment:
Some info, such as taxes or shipping, might be empty in the cart. I thought it would be more convenient to allow empty prices in the base price than checking every time to output a 0.